packages/spatial/src/renderer/functions/configureEffectComposer.ts

```diff
@@ -84,8 +85,32 @@ export const configureEffectComposer = (
 
   const effects: any[] = []
 
+  const performanceTier = getState(PerformanceState).tier
+
+  let smaaPreset
+  switch (performanceTier) {
```

Comment:
this is nice, we should see if we can also update this property dynamically in a reactor without recreating the entire postprocessing pipeline